daphne_worker/src/config.rs

```diff
@@ -360,6 +360,8 @@ impl DaphneWorkerConfig {
     }
 }
 
+pub(crate) type HpkeConfigs = HashMap<u8, HpkeReceiverConfig>;
```

Comment:
nit: `HpkeConfigs` is slightly difficult to visually distinguish from `HpkeConfig`. (I had some type confusion while reviewing.) A more distinguished name would be useful.

Reply:
would `HpkeKeySet` be a good name? :thinking: I don't like the name I picked either :sweat_smile:

Reply:
How about `HpkeRecieverConfigSet`? It's a little verbose but it is consistent.